grpc_test.go

```diff
@@ -27,8 +27,25 @@ import (
 
 type s struct{}
 
+var lcFailed = false
+
+type errorer struct {
+	t *testing.T
+}
+
+func (e errorer) Errorf(format string, args ...interface{}) {
+	lcFailed = true
```

Comment:
This might be too fancy: atomic? So we can run subtests in parallel?

Reply:
It's easy enough to do, anyway.  Done.